src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User approvedUser = userService.findByEmail(email);
+        if (approvedUser != null && password.equals(approvedUser.getPassword())) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
also some minor optimization :wink: 
```suggestion
        return approvedUser != null && password.equals(approvedUser.getPassword());
```